frontend/admin/src/js/components/pool/EditPool/OtherRequirementsSection.tsx

```diff
@@ -9,6 +9,7 @@ import {
   getLanguageRequirement,
   getSecurityClearance,
 } from "@common/constants/localizedConstants";
+import { notEmpty } from "@common/helpers/util";
```

Comment:
Unused imports `isEmpty` and `notEmtpy`

Reply:
Fixed with [remove unused imports](https://github.com/GCTC-NTGC/gc-digital-talent/pull/3644/commits/1fead82973ea81de67acd27a35ecc19cb3806fad)